plugins/modules/k8s.py

```diff
@@ -172,6 +172,18 @@
         - When set to True, server-side apply will force the changes against conflicts.
         type: bool
         default: False
+  delete_all:
+    description:
+    - When this option is set to I(true) and I(state=absent),
+      module will delete on all resources, in the namespace of the specified resource type.
+    - Ignored when C(state) is not set to I(absent) or when one of (src),
+      C(name)  or C(resource_definition) is provided.
+    - Parameter C(kind) is required to use this option.
```

Comment:
It's probably worth adding a note here that this can be used with the `label_selectors` param. I suspect this is something a lot of people will want to do.

Reply:
updated